vhd/packer/install-dependencies.sh

```diff
@@ -340,10 +340,9 @@ K8S_VERSIONS="
 1.16.7-azs
 1.16.6
 1.16.6-azs
+1.15.11
```

Comment:
@mboersma skip the VHD changes, let's do that in a separate PR: https://github.com/Azure/aks-engine/pull/2894

We'll merge the above PR after all new k8s version PRs have landed in master